politeiawww/user/mysqldb/log.go

```diff
@@ -1,3 +1,7 @@
+// Copyright (c) 2021 The Decred developers
```

Comment:
If you copy code from another repo, which is the case here, you need to keep the existing copyright statement.